_posts/2021-02-23-winter-update.md

```diff
@@ -0,0 +1,155 @@
+but we do track them and consider your feedback in our planning process!
+
+## Autumn 2020 (October to December) announcements
+
+The end of 2020 continued to test our adaptability and resiliency.
+For the third quarter in a row,
+the training and community team found ourselves relocated to a different group at the Hutch.
+**The Coop team (Kate Hertweck and Lauren Wolfe) is excited to have joined the Bioinformatics and Genomics [Shared Resources](https://www.fredhutch.org/en/research/shared-resources/core-facilities/genomics-bioinformatics.html) in November 2020 as a part of the newly emerging Hutch Data Core.**
+The group with which we had previously been affiliated,
+Scientific Computing,
+moved to Center IT,
+though we continue to communicate regularly and work on collaborative projects with them.
+
+Some of you may not be familiar with [Shared Resources](https://sharedresources.fredhutch.org),
+especially if your job focuses exclusively on data analysis and/or software development.
+Shared Resources is a collection of core facilities and laboratories available to researchers for performing experiments and generating data. 
+Our team reports to Jeff Delrow,
+the Senior Director of Shared Resources' Molecular and Cellular Scientific Resources,
+and the head of the Genomics and Bioinformatics Team.
+We've spent the last few months learning more about how Shared Resources currently supports training and outreach for both data generation and analysis,
+and thinking about ways that we can share information about and contibute to these efforts.
+
+## 2020 Year in Review summary
+
+Given the number of adjustments to our team's placement at the Hutch,
+we spent a lot of time this year adjusting to work with new teams.
+While this presented challenges associate with logistics, communication, and planning,
+it also allowed the opportunity for us to learn a lot about how different groups at the Hutch work to support data-intensive research.
+
+Here are some of the major accomplishments for the last year:
+
+**fredhutch.io course teaching and deployment**
+
+We taught 14 classes through fredhutch.io during 2020,
+10 of which were taught online using MS Teams.
+We developed [guidelines and suggestions](https://fredhutchio.github.io/instructors/)
+for teaching computational courses remotely to support our instructors needing to make this shift.
+
+We've recognized the way that people value and interact with data
+has shifted with so much of our community working remotely,
+and have adjusted our methods of supporting training to match these approaches.
+
+While the materials we use for teaching have always been publibly available (online through GitHub repositories),
+we converted our classes to accessible, fully-rendered websites
+that are available for reference during class,
+as well as for self-led, work-at-your-own-pace learning.
+These materials include:
+- [Intro to R](https://github.com/fredhutchio/r_intro)
```

Comment:
just wondering if it makes sense to link to the fully-rendered website? If not, maybe we should make the link to the website more visible in the README. I don' think a lot of folks think to look for the link on the sidebar.

Reply:
You're absolutely right; can you update those links? I thought those were the ones I was grabbing

Reply:
yup! on it!